tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Service/Microsoft.Legal.MatterCenter.Repository/SharePointInfo/Taxonomy.cs

```diff
@@ -228,12 +228,12 @@ private TaxonomyResponseVM GetTaxonomyHierarchy (TermStore termStore, TermStoreD
 
 
         /// <summary>
-        /// Gets the practice group term set hierarchy with generic code using JSONWriter
+        /// Gets the managed term set hierarchy with generic code using JSONWriter
         /// </summary>
         /// <param name="termSet">Term set object holding Practice Group terms</param>
         /// <param name="termStoreDetails">Term Store object containing Term store data</param>
         /// <returns>Serialized Object of Term Set</returns>
-        private string GetPracticeGroupTermSetHierarchy(ClientContext clientContext, TermSet termSet, TermStoreDetails termStoreDetails)
+        private string GetManagedTermSetHierarchy(ClientContext clientContext, TermSet termSet, TermStoreDetails termStoreDetails)
```

Comment:
Changed the method name from GetPracticeGroupTermSetHierarchy to GetManagedTermSetHierarchy